models/contact.js

```diff
@@ -0,0 +1,46 @@
+const { Schema, model } = require("mongoose");
+const Joi = require("joi");
+
+const { handleMongooseError } = require("../helpers");
+
+const contactSchema = Schema({
+  name: {
+    type: String,
+    required: [true, "Set name for contact"],
+  },
+  email: {
+    type: String,
+  },
+  phone: {
+    type: String,
+  },
+  favorite: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+contactSchema.post("save", handleMongooseError);
+
+const addSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string(),
```

Comment:
You should validate email and phone properly. It shouldnt be just strings

Reply:
Ok